cdc/owner.go

```diff
@@ -494,18 +493,7 @@ func (o *ownerImpl) calcResolvedTs() error {
 		minCheckpointTs := cfInfo.TargetTs
 
 		if len(cfInfo.tables) == 0 {
```

Comment:
after this change when there no tables, the resolve ts can be forward and the checkpoint ts can not be forward?

Reply:
yes, in fact if there is no tables, the `checkpoint ts` is meaningless until cdc meets an add table DDL.